Gruntfile.js

```diff
@@ -351,6 +351,15 @@ module.exports = function (grunt) {
             test : {
                 files: ['<%= meta.test %>', 'test/**/*'],
                 tasks: ['eslint:test']
+            },
+            bramble : {
+                files: [
+                    'src/bramble/client/**/*.js',
+                    'src/bramble/thirdparty/**/*.js',
+                    'src/bramble/ChannelUtils.js',
+                    'thirdparty/filer/dist/filer.min.js'
+                ],
+                tasks: ['requirejs:iframe']
```

Comment:
let's instead change the task to be `grunt build-browser` as it should only build the iframe api if there were changes to those files and `grunt build-browser` should handle that.

Reply:
@gideonthomas 
The `build-browser` takes too much time to process. If we only want to compiler the bramble.js file. We only need to run `requirejs:iframe`. See this link.
https://github.com/mozilla/brackets/blob/master/Gruntfile.js#L226-L255

The purpose of this PR is that we can reduce build time by running `requirejs:iframe` instead of `build-browser`.
What do you think?

Reply:
sorry, I meant just change it to `grunt build-browser-dev`, that should only run a single task and should be fairly quick.